app/components/graph-editor/README.md

```diff
@@ -0,0 +1,82 @@
+# Notes
```

Comment:
Really nice documentation. We should try and emulate these kinds of READMEs throughout the project! However, we should probably have a link from the main README to this README?